packages/integration/src/bitbucket/config.ts

```diff
@@ -86,14 +96,35 @@ export function readBitbucketIntegrationConfig(
     );
   }
 
+  if (baseUrl) {
+    baseUrl = trimEnd(baseUrl, '/');
+  } else {
+    baseUrl = `https://${host}`;
+  }
+
+  if (!isValidUrl(baseUrl)) {
+    throw new Error(
+      `Invalid Bitbucket integration config, '${baseUrl}' is not a valid baseUrl`,
+    );
+  }
+
   if (apiBaseUrl) {
     apiBaseUrl = trimEnd(apiBaseUrl, '/');
   } else if (host === BITBUCKET_HOST) {
     apiBaseUrl = BITBUCKET_API_BASE_URL;
+  } else {
+    apiBaseUrl = `${baseUrl}/rest/api/1.0`;
```

Comment:
While we can infer the `apiBaseUrl` using the `baseUrl`, the real question is, should we? 
Could this value ever be different, or does this suffice as a default fallback?